tests/utils/test_command_converter.py

```
@@ -0,0 +1,259 @@
+import pytest
+import asyncio
```

Flake8 (7.2.0)

[error] 'asyncio' imported but unused

(F401)